lib/web/fetch/body.js

```diff
@@ -264,14 +277,18 @@ function safelyExtractBody (object, keepalive = false) {
   return extractBody(object, keepalive)
 }
 
-function cloneBody (body) {
+function cloneBody (instance, body) {
   // To clone a body body, run these steps:
 
   // https://fetch.spec.whatwg.org/#concept-body-clone
 
   // 1. Let « out1, out2 » be the result of teeing body’s stream.
   const [out1, out2] = body.stream.tee()
 
+  if (hasFinalizationRegistry) {
+    streamRegistry.register(instance, new WeakRef(out1))
```

Comment:
is out1 the new stream and out2 the old stream?

Reply:
No, out1 is set to the 'old' request body and out2 is the new one, but from testing only this is needed, at least for the cases tested. Maybe v8 is smart enough to realize that out2 can be collected once out1 is?